aiohttp/_websocket/models.py

```diff
@@ -59,6 +59,21 @@ def json(
         return loads(self.data)
 
 
+class WSMessageTextBytes(NamedTuple):
+    """WebSocket TEXT message with raw bytes (no UTF-8 decoding)."""
+
+    data: bytes
+    size: int
+    extra: str | None = None
+    type: Literal[WSMsgType.TEXT] = WSMsgType.TEXT
+
+    def json(
+        self, *, loads: Callable[[str | bytes | bytearray], Any] = json.loads
```

Comment:
This only needs to accept bytes, right? As we're passing self.data in.
```suggestion
        self, *, loads: Callable[[bytes], Any] = json.loads
```